payment_alipay/__openerp__.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Alipay Payment Acquirer',
+    'category': 'Hidden',
+    'summary': 'Payment Acquirer: Alipay Implementation',
+    'version': '8.0.2.0.1',
```

Comment:
@elicoidal It's for V8,haven' been migrated.And should I create a new one for branch 8.0?